dvol_python/testtools.py

```diff
@@ -20,6 +22,17 @@
     "Not expected to work in Python version"
 )
 
+def _skip_max_docker_ver(ver):
+    try:
+        return max_ver(DOCKER_VERSION, ver) == ver
+    except ValueError:
+        return False
+
+skip_if_docker_version_less_than = lambda ver: skipIf(
+    _skip_max_docker_ver(ver),
```

Comment:
I think I'm reading this right... If `DOCKER_VERSION` is equal to `ver` then `_skip_max_docker_ver` will return `True` which means that the test will be skipped. But we only want to skip if `DOCKER_VERSION` is _less_ than `ver`.